hack/preview-template.env

```diff
@@ -24,6 +24,15 @@ export MY_GITHUB_TOKEN=
 
 # Optional
 
+## Pipeline Service pointers
+### Pipeline Service workspace, default $ROOT_WORKSPACE:pipeline-service
+### On kcp-stable - root:redhat-pipeline-service-compute
+export PIPELINE_SERVICE_SP_WORKSPACE=
```

Comment:
ok, now I see that it was mentioned here, but I would love to use the default values that correspond to the production ones.
BTW, don't forget that create-user-workspace.sh doesn't load values from this file. you should be able to use it without preview mode - in CPS using the APIExports from `root`